packages/client/src/components/app/App.tsx

```diff
@@ -31,6 +32,7 @@ interface Props {
 
 function App({ client }: Props): ReactElement {
   const classes = useStyles();
+  const [trainerState, setTrainerState] = useState(9);
```

Comment:
Instead of using a React State, can this be an Apollo/GraphQL state so it updates the data?